clusterloader2/testing/watch-list/config.yaml

```diff
@@ -0,0 +1,62 @@
+{{$testDuration := "5m"}}
+name: huge-secrets
```

Comment:
I personally wouldn't call this test "huge-secrets", because it's not what it is really testing.
We're really testing streaming lists.

So maybe "streaming-list"? Or "watch-list" ?

Reply:
okay, will change to `watch-list`